src/Illuminate/Foundation/Console/ViewMakeCommand.php

```diff
@@ -0,0 +1,195 @@
+use Illuminate\Support\Str;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+
+class ViewMakeCommand extends GeneratorCommand
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'make:view';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Create a new view';
+
+    /**
+     * The type of class being generated.
+     *
+     * @var string
+     */
+    protected $type = 'View';
+
+    /**
+     * Determine if the class already exists.
+     *
+     * @param  string $rawName
+     * @return bool
+     */
+    protected function alreadyExists($rawName)
+    {
+        return file_exists($this->getPath($this->parseName($rawName)));
+    }
+
+    /**
+     * Parse the name and format according to the root namespace.
+     *
+     * @param  string $name
+     * @return string
+     */
+    protected function parseName($name)
+    {
+        if (Str::contains($name, '\\')) {
+            $name = str_replace('\\', '/', $name);
+        }
+        if (Str::contains($name, '.')) {
+            $name = str_replace('.', '/', $name);
+        }
+
+        return $name;
+    }
+
+    /**
+     * Get the stub file for the generator.
+     *
+     * @return string
+     */
+    protected function getStub()
+    {
+        return __DIR__.'/stubs/view.stub';
+    }
+
+    /**
+     * Get the destination class path.
+     *
+     * @param  string $name
+     * @return string
+     */
+    protected function getPath($name)
+    {
+        return resource_path('/views/'.$name.'.blade.php');
+    }
+
+    /**
+     * Build the class with the given name.
+     *
+     * @param  string $name
+     * @return string
+     */
+    protected function buildClass($name)
+    {
+        $stub = $this->files->get($this->getStub());
+        $parent = $this->option('parent');
+        $section = $this->option('section');
+        $class = $this->option('class');
+        $stacks = $this->option('stacks');
+        $this->replaceParentView($stub, $parent)->replaceSection($stub, $section, $class)->insertStacks($stub, $stacks);
+
+        return $stub;
+    }
+
+    /**
+     * Replace the Parent View Name for the given stub.
+     *
+     * @param  string $stub
+     * @param  string $parentViewName
+     * @return $this
+     */
+    protected function replaceParentView(&$stub, $parentViewName)
+    {
+        $stub = str_replace(
+            'DummyParentView', $parentViewName, $stub
+        );
+
+        return $this;
+    }
+
+    /**
+     * Replace the Section Name for the given stub.
+     *
+     * @param  string $stub
+     * @param  string $sectionName
+     * @param  string $class
+     * @return $this
+     */
+    protected function replaceSection(&$stub, $sectionName, $class)
+    {
+        $stub = str_replace(
+            'DummySection', $sectionName, $stub
```

Comment:
On every other command it is on 3 lines.

Example: https://github.com/laravel/framework/blob/5.3/src/Illuminate/Foundation/Console/ListenerMakeCommand.php#L62